packages/gatsby-plugin-sharp/src/index.js

```diff
@@ -411,19 +420,33 @@ async function notMemoizedbase64({ file, args = {}, reporter }) {
     return null
   }
 
+  // Not all tranformer plugins are going to provide this parameter hence the fallback
```

Comment:
One thing I like to do (and more of a house-keeping thing than anything substantive) is to calculate the key before using it in cache.get or cache.set, e.g.

```js
const cacheKey = base64CacheKey(file.internal.contentDigest, options);

if (cache) {
  const cached = await cache.get(cacheKey)
  if (cached) {
    return cached;
  }
}
```